src/pages/index.js

```diff
@@ -264,6 +136,20 @@ HomePage.propTypes = {
         contentPadding: PropTypes.string,
       }),
     }),
+    allMarkdownRemark: PropTypes.shape({
+      edges: PropTypes.arrayOf(
+        PropTypes.shape({
+          whatsNewPosts: PropTypes.shape({
+            frontMatter: PropTypes.shape({
+              title: PropTypes.string,
+            }),
+            fields: PropTypes.shape({
+              slug: PropTypes.string,
+            }),
+          }),
+        })
+      ),
```

Comment:
I ran the query in my local version of GraphQL and I got the following response:

```json
{
  "data": {
    "allMarkdownRemark": {
      "edges": [
        {
          "node": {
            "id": "80d638c5-d1e2-58c3-b4d7-98c3a8581cb2",
            "frontmatter": {
              "title": "Kubernetes Horizontal Pod Autoscaler (HPA) support"
            },
            "fields": {
              "slug": "/whats-new/2021/11/Whats-New-K8sHPA"
            }
          }
        },
```

Which makes me think that, if we're going to type this thing all the way down, we want the types to be something like this (`whatsNewPosts` > `node` and `frontMatter` to `frontmatter`):

```suggestion
      edges: PropTypes.arrayOf(
        PropTypes.shape({
          node: PropTypes.shape({
            frontmatter: PropTypes.shape({
              title: PropTypes.string,
            }),
            fields: PropTypes.shape({
              slug: PropTypes.string,
            }),
          }),
        })
      ),
```